util/topgen/intermodule.py

```diff
@@ -387,16 +387,26 @@ def elab_intermodule(topcfg: OrderedDict):
         sig['external'] = True
 
         sig_name = port if port != "" else intersignal_format(sig)
-        sig["top_signame"] = sig_name
+
+        # if top signame already defined, then a previous connection cateogry
```

Comment:
nitnit: cateogry -> category

Reply:
o yes you're right. Originally I was going to use this in the dictionary construction below, but doesn't seem like i need that anymore.